helm-chart/images/jupyterhub-ssh/jupyterhub_ssh_config.py

```diff
@@ -7,3 +10,4 @@
 c.JupyterHubSSH.host_key_path = "/etc/jupyterhub-ssh/config/hostKey"
 c.JupyterHubSSH.hub_url = config["hubUrl"]
 c.JupyterHubSSH.debug = True
+c.JupyterHubSSH.ssh_conn_timeout = int(sshConnTimeout)
```

Comment:
I think you can read this directly from the values.yaml file that has been parsed as a Python dictionary from the mounted YAML file. By doing that, you avoid needing to use the int conversion as well.

```suggestion
c.JupyterHubSSH.start_timeout = config["ssh"]["start_timeout"]
```